src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -140,6 +147,11 @@ public IntakeSubsystem() {
 		indexSensor = new DigitalInput(INDEX_SENSOR);
 		feederSensor = new DigitalInput(FEEDER_SENSOR);
 
+		intakeFrontSensor = intakeMotorFront.getForwardLimitSwitch(SparkLimitSwitch.Type.kNormallyOpen);
+		intakeBackSensor = intakeMotorBack.getForwardLimitSwitch(SparkLimitSwitch.Type.kNormallyOpen);
+		intakeLeftSensor = intakeMotorLeft.getForwardLimitSwitch(SparkLimitSwitch.Type.kNormallyOpen);
+		intakeRightSensor = intakeMotorRight.getForwardLimitSwitch(SparkLimitSwitch.Type.kNormallyOpen);
```

Comment:
Also need to update the motor configuration code so the controller knows the limit switch is plugged in (the same wire can also be used for an encoder), and to not stop the motor when the switch is signaled

Reply:
We updated the configuration but not sure if it'll impact the motor

Reply:
enableLimitSwitch() is set to false so it shouldn't impact the motors